src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -25,6 +30,7 @@
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
         // TODO: add log message about successful fetched data from DB
+        logger.info("The information successful fetched from DB by user: {}", userId);
```

Comment:
The log message states that data was fetched from the DB, but the method actually returns dummy data. Consider adjusting the log message to reflect that this is a test implementation and not actual DB fetching.